### ISSUE TITLE
Deleted files still uses storage

### ISSUE THREAD

**Issue by 123321mario** [lmirel/fm_psx#1]
Hello, so I use my PS3 Slim (with HEN on last firmware) as a Blu-Ray ripper with MultiMan and I use the Simple file manager to copy the iso that MultiMan makes to my NTFS USB hard drive. The copy works fine no problem but when it's copied I delete the file on the PS3 to save space, but when I delete with simple file manager the file is gone but it doesn't free storage space, like if the file was still here! The only way to fix that is basically reinstall the whole PS3 system ! If you have any questions about my setup please ask me!

**Comment by lmirel** [comment]
sorry to hear that. the simple file manager tool was written as a demo for implementing ExFat support and I’m not maintaining it.
it was integrated into IRISMAN https://github.com/aldostools/IRISMAN/releases which I recommend if you need a complete file manager and more. I think he provides the filemanager as standalone too.
anyway, I don’t remember of an issue as you’re describing but it could be related to HEN which I never tested.
anyway, instead of reinstalling the whole ps3 system, you should try just rebuilding the database, via recovery menu, maybe that works. sorry I can’t help more

**Comment by lmirel** [comment]
I also remember that Multiman has NTFS support too. any reason you aren’t using that one instead? again, I have no experience with HEN so I may be asking stupid questions. I also don’t use Multiman since Irisman can do everything I need.

**Comment by 123321mario** [comment]
Thanks for answering, I will try that hoping it works and don't worry i don't have any important data on the system, a reinstall isn't a big deal :)

**Comment by 123321mario** [comment]
> I also remember that Multiman has NTFS support too. any reason you aren’t using that one instead? again, I have no experience with HEN so I may be asking stupid questions. I also don’t use Multiman since Irisman can do everything I need.

I don't think it supports ntfs, the drive isn't showing in the "save as" window when i make the iso

**Comment by lmirel** [comment]
I think it can only transfer files to/from NTFS but not use it as active storage. also, you need to ‘activate’ the NTFS support from the options menu if I remember well. then there is also the option of using FTP from a PC but the existing implementation is rather slow from my experience. yours may vary.
